doc/conf.py

```diff
@@ -58,9 +58,15 @@
     'sphinx.ext.autosectionlabel',
     'numpydoc',
     'sphinx.ext.linkcode',
-    'gh_substitutions',  # custom ext, see ./sphinxext/gh_substitutions.py
+    'gh_substitutions',
+    'myst_parser',
+    # custom ext, see ./sphinxext/gh_substitutions.py
```

Comment:
can you keep this on the gh_substitutions line (it's a bit mysterious so it helps to know it's a custom one)